tools/ci_build/build.py

```diff
@@ -311,7 +311,7 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
         os.makedirs(config_build_dir, exist_ok=True)
 
         if args.use_tvm:
-            os.environ["PATH"] += os.pathsep + os.path.join(config_build_dir, "external", "tvm", config)
```

Comment:
Why it must be at the front?

Reply:
We build using this command "--config Debug Release". So first the debug path of tvm.dll is appended to the path followed by the release path. Now when the debug test binary runs, all is good since it's able to find the corresponding path (debug path) of tvm.dll. When the release test binary run, the same debug path of tvm.dll is used since it occurs first in the list as a result of which the release test crashes. This fix ensure the right path is at the front.

Reply:
Then we should remove this line. PATH can be set in run_subprocess() function